great_hall.py

```diff
@@ -0,0 +1,39 @@
+class GreatHall:
+    SOUTH = {
+        "message":"You head south toward the vestibule.",
+        "location":3,
+    }
+    EAST = {
+        "message":"You walk toward the east wing of the manor.  You feel as though the eyes of the people in the portrait follow you.",
+        "location":9,
+    }
+    WEST = {
+        "message":"You walk to the west wing of the manor, the old floor creaking beneath your feet",
+        "location":5
+    }
+    UP = {
+        "message":"You walk past the portraits and up the carpeted steps to the second floor.",
+        "location":18,
+    }
+    commands = {
+        "EXAMINE PAINTINGS":"You see portraits of a distinguished-looking man and a pale-skinned woman.",
+        "EXAMINE MIRROR":"You can see yourself.",  # TODO: add{wearing a smoking jacket/a wolf pelt} if they are in the inventory
+        "SOUTH":SOUTH,
+        "EXIT SOUTH":SOUTH,
+        "GO SOUTH":SOUTH,
+        "EAST":EAST,
+        "EXIT EAST":EAST,
+        "GO EAST":EAST,
+        "WEST":WEST,
+        "EXIT WEST":WEST,
```

Comment:
Think about how you can create a class for these commands so you can have one command with a list of acceptable inputs instead of multiple strings in this dictionary.